client/idrepo/console/src/main/java/org/apache/syncope/client/console/pages/Realms.java

```diff
@@ -197,8 +197,8 @@ protected void onClickTemplate(final AjaxRequestTarget target) {
         @Override
         protected void setWindowClosedReloadCallback(final BaseModal<?> modal) {
             modal.setWindowClosedCallback(target -> {
-                if (modal.getContent() instanceof ResultPage) {
-                    Serializable result = ResultPage.class.cast(modal.getContent()).getResult();
+                if (modal.getContent() instanceof ResultPanel) {
+                    Object result = ResultPanel.class.cast(modal.getContent()).getResult();
```

Comment:
Why not `Serializable` any more?

Reply:
I use this panel like this `AbstractResultPanel<String, List<StatusBean>>` and List is not `Serializable`.

Reply:
Mmmh ok.
Please ensure that neither Core nor Console logs show serialization errors while using such panel.

Reply:
going to find a workaround

Reply:
Ok, let me check

Reply:
Going to apply the same solution as in https://github.com/apache/syncope/blob/master/client/idm/console/src/main/java/org/apache/syncope/client/console/commons/IdMAnyDirectoryPanelAdditionalActionsProvider.java#L109